package/MDAnalysis/auxiliary/base.py

```diff
@@ -431,6 +435,74 @@ def read_ts(self, ts):
             self._add_step_to_frame_data(ts.time)
         self.frame_rep = self.calc_representative()
 
+    def attach_auxiliary(self,
+                         coord_parent,
+                         aux_spec: Union[str, Dict[str, str]],
+                         format: Optional[str] = None,
+                         **kwargs) -> None:
+        """ This method is called from within
+            :func:`MDAnalysis.coordinates.base.add_auxiliary()`.
+            `add_auxiliary` should be agnostic to the type of AuxReader, so the
+            method call leads here instead. First, some checks are done on
+            the arguments to make sure the input is treated properly. Then, 
+            the AuxReader(s) with appropriate :attr:`data_selector` are
+            associated with the `coord_parent` from which `add_auxiliary` was
+            called. """
```

Comment:
weird formatting/indentation, please make it fit the rest of the code

Reply:
Make it a standard doc string (following our doc string standards https://userguide.mdanalysis.org/stable/contributing_code.html#guidelines-for-writing-docstrings